docs/index.yml

```diff
@@ -0,0 +1,252 @@
+### YamlMime:Hub
+
+title: .NET documentation
+summary: Learn to use .NET to create a variety of applications on any platform using C#, Visual Basic, and F#. Browse API reference, sample code, tutorials, and more.
+brand: dotnet
+
+metadata:
+  title: .NET documentation
+  description: Learn to use .NET to create a variety of applications on any platform using C#, Visual Basic, and F#. Browse API reference, sample code, tutorials, and more.
+  ms.product: dotnet
+  ms.topic: hub-page
+  ms.collection: collection
+  author: BillWagner
+  ms.author: wiwagn
+  ms.date: 10/29/2019
+
+# highlightedContent section (optional)
+# Maximum of 8 items
+highlightedContent:
+# itemType: architecture | concept | deploy | download | get-started | how-to-guide | learn | overview | quickstart | reference | tutorial | video | whats-new
+  items:
+    # Card
+    - title: "C# introduction. A quick interactive start"
+      itemType: get-started
+      url: csharp/tutorials/intro-to-csharp/hello-world.yml
+    # Card
+    - title: "Download the .NET SDK"
+      itemType: download
+      url: https://dotnet.microsoft.com/download
+    # Card
+    - title: "Build your first .NET Core app. Learn how to build a .NET core console app"
+      itemType: get-started
+      url: core/get-started.md
+    # Card
+    - title: "Build your first web app. Learn how to build your first ASP.NET Core web app"
+      itemType: learn
+      url: https://dotnet.microsoft.com/learn/aspnet/hello-world-tutorial/intro
+
```

Comment:
Why do we not link to the ASP.NET Core Get started tutorial (https://docs.microsoft.com/en-us/aspnet/core/getting-started/?view=aspnetcore-3.0), instead of the dotnet site Get started tutorial for the highlighted content section?

Reply:
@mcleblanc That's the current link on the [get-started pivot](https://docs.microsoft.com/en-us/dotnet/#pivot=docs&panel=getstarted). I'll change that in the next PR. 

I'd like the first commit on this branch to just convert to the new format. Then, we'll have a history of all the hub destination changes.